core/src/main/java/com/group5/core/screens/MainGameScreen.java

```diff
@@ -125,9 +147,14 @@ public void render(final float delta) {
 
         if (!gameOverMenuActive && !(worldManager.getGameStatus())) {
             gameOverMenuActive = true;
-            stage.getActors().get(0).setVisible(true);
+            gameOverTable.setVisible(true);
+            gameOverScore.setText("Score:" + Integer.toString(Math.round(score)));
+            scoreLabel.setVisible(false);
             Gdx.input.setInputProcessor(stage);
         }
+        score = score + delta;
```

Comment:
In the requirements the scoring is connected to distance traveled. If chicken during the course of a game would start to run faster, the scoring and distance traveled won't be the same.